Lib/test/test_site.py

```diff
@@ -181,9 +181,6 @@ def test_addsitedir(self):
         finally:
             pth_file.cleanup()
 
-    def test_getuserbase(self):
-        self.assertEqual(site._getuserbase(), sysconfig._getuserbase())
```

Comment:
Instead of removing this test, can you move the assertion to the remaining test_userbase?

Reply:
Actually, it turns out, this tests a different function than the other `test_getuserbase`. So I have a better idea...